doc/classes/AudioListener3D.xml

```diff
@@ -4,7 +4,7 @@
 		Overrides the location sounds are heard from.
 	</brief_description>
 	<description>
-		Once added to the scene tree and enabled using [method make_current], this node will override the location sounds are heard from. This can be used to listen from a location different from the [Camera3D].
+		Once added to the scene tree and enabled using [method make_current], this node will override the location sounds are heard from. This can be used to listen from a location different from the [Camera3D]. The first [AudioListener3D] added to the scene tree will automatically become current, regardless of [member current] property.
```

Comment:
There's no `current` property, only the getter and setter (for some reason)

Reply:
Lies
![image](https://github.com/user-attachments/assets/2c923ab7-f5b4-4584-959b-24ddcba2b2b9)
For whatever reason it's defined with `_get_property_list()` .-.

Reply:
I'm guessing it uses `get_property_list()` because `is_current` may not always return the same value as what `set_current()` was used with, since multiple AudioListener3D nodes can overtake each other.